README.md

```diff
@@ -3,15 +3,17 @@
-* [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
-* [lemmy](https://github.com/dessalines/lemmy) : A federated alternative to reddit in rust.
-* [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
-* [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
-* [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
-* [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container. 
+
+- [kriry.com](http://kriry.com/) : Explore-Interactive net. [source](https://github.com/kriry/waler)
+- [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
+- [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
+- [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
+- [lemmy](https://github.com/dessalines/lemmy) : A federated alternative to reddit in rust.
+- [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
+- [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
+- [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
+- [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container.
+- [Complete Actix 2.x REST Server](https://github.com/ddimaria/rust-actix-example): Actix 2.x HTTP Server, Multi-Database Support (CockroachDB, Postgres, MySQL, Sqlite), JWT Support, Async Caching Layer with a Simple API, Public and Secure Static File Service, Diesel Database Operations are Non-Blocking, Filesystem Organized for Scale, .env for Local Development, Integrated Application State with a Simple API, Lazy Static Config struct, Built-in Healthcheck (includes cargo version info), Listeners configured for TDD, Custom Errors and HTTP Payload/Json Validation, Secure Argon2i Password Hashing, CORS Support, Unit and Integration Tests, Test Coverage Reports, Dockerfile for Running the Server in a Container, and TravisCI Integration.
```

Comment:
I think this description should be shorter. Maybe just call it something like 'feature-full' or 'production-like' example and add some more general ideal of what it offers, like DB support and so on. People looking for examples could get more details when checking the linked repo.

Reply:
I significantly shortened it.  Let me know if you want it shorter.  Thanks!